lib/color_guard/views/all_features.html.erb

```diff
@@ -0,0 +1,70 @@
+<h2>URL flags</h2>
+<code>
+  URL flags can be set by adding parameters to the URL with the 'color_guard_' prefix<br />
+  <strong>?color_guard_party=true&amp;color_guard_lame_stuff=false</strong>
+</code>
+<table>
+  <thead>
+    <tr>
+      <th>Feature</th>
+      <th>Percentage</th>
+    </tr>
+  </thead>
+  <tbody>
+    <% @all_features[0].each do |key, value| %>
```

Comment:
I don't like this UI assuming that there is one store with three inner stores as that's not necessarily true. The color guard middleware can be configured in multiple different ways and I don't think this UI should crash if the currently configured store doesn't match the one in the example. To me fixing this seems related to my comment on the return type of `Store::Stack#all`